celery/canvas.py

```diff
@@ -1296,8 +1304,8 @@ def app(self):
         return app if app is not None else current_app
 
 
-@Signature.register_type()
-class chord(Signature):
+@Signature.register_type(name="chord")
+class _chord(Signature):
```

Comment:
Why the name change?

Reply:
This is so we can touch the `chord` class (as `_chord`) from methods in `group` which shadow the class' name with their kwargs (e.g. `group._apply_tasks(..., chord=None, ...)`)